Documentation/Index.md

```diff
@@ -1178,12 +1178,12 @@ Any object that can be encoded and decoded can be stored as a blob of data in SQ
 We can create an `NSData` bridge rather trivially.
 
 ``` swift
-extension NSData: Value {
+extension NSData{
```

Comment:
Confirming to the value protocol throws an error in Swift 2.2 and up. Says
you can't redefine the inheritance

On Tuesday, July 19, 2016, Nick Shelley <notifications@github.com> wrote:

> In Documentation/Index.md
> <https://github.com/stephencelis/SQLite.swift/pull/471#discussion_r71406531>
> :
>
> > @@ -1178,12 +1178,12 @@ Any object that can be encoded and decoded can be stored as a blob of data in SQ
> >  We can create an `NSData` bridge rather trivially.
> >
> >  ``` swift
> > -extension NSData: Value {
> > +extension NSData{
>
> This should still conform to the Value protocol
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/stephencelis/SQLite.swift/pull/471/files/6d2da4ca2fe27784cdf644e24ba6a8a4ba53822b#r71406531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD2KiT2vp0v1oc3TY01_w4gSfIUF-yy0ks5qXST-gaJpZM4JPat->
> .
>